core/commonJs/src/LocalDate.kt

```diff
@@ -42,6 +42,13 @@ public actual class LocalDate internal constructor(internal val value: jtLocalDa
             throw e
         }
 
+        internal actual fun fromEpochDays(epochDays: Long): LocalDate {
+            require(epochDays in Int.MIN_VALUE..Int.MAX_VALUE) {
+                "Invalid date: boundaries of LocalDate exceeded"
+            }
+            return fromEpochDays(epochDays.toInt())
+        }
```

Comment:
```suggestion
        internal actual fun fromEpochDays(epochDays: Long): LocalDate =
            fromEpochDays(epochDays.clampToInt())
```
Our internal `clampToInt` function clamps the `Long` to `Int.MAX_VALUE` or `Int.MIN_VALUE`, and both of these values don't fit into the range checked by `fromEpochDays(Int)`.

Reply:
> both of these values don't fit into the range checked by fromEpochDays(Int)

We can assume this to be true because we know how `fromEpochDays(Int)` is implemented, but the signature itself does not require that to be the case. Do we mind that we're relying on that assumption?

Reply:
`fromEpochDays(Long)` already has the knowledge that days outside `Int` won't work on this platform, and learning the same about `Int.MAX_VALUE` is just the next step.